### PR TITLE
feat: 🎸 user can update clusterURL when controller unsupported

### PR DESCRIPTION
**Pull request by cameronperera**
## Description

user can update clusterURL when controller unsupported

## Screenshots (if appropriate):

https://github.com/hashicorp/boundary-ui/assets/29386339/c6840d70-6cc8-45ee-a788-153a68f4e307

## How to Test

Start the boundary desktop client and connect it to a version of boundary that does not support pagination (<=0.14.3). When you start the desktop client, the `unsupported controller` modal should appear with a new button to redirect the user to the cluster url page to update it if they need to.

## Checklist:

- [x] I have added before and after screenshots for UI changes
- [ ] ~I have added JSON response output for API changes~
- [x] I have added steps to reproduce and test for bug fixes in the description
- [ ] ~I have commented on my code, particularly in hard-to-understand areas~
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by dvonanderson — APPROVED** [review]
LGTM

File: ui/desktop/app/templates/scopes.hbs [hunk 1]
```diff
@@ -48,6 +48,13 @@
...
+      <Hds::Button
+        type='button'
+        @color='secondary'
+        @route='cluster-url'
+        @text={{t 'unsupported-controller.buttons.change-cluster-url'}}
+        data-test-change-cluster-url
+      />
     </M.Footer>
   </Hds::Modal>
 {{/unless}}
```
Comment (lisbet-alvarez) [inline ui/desktop/app/templates/scopes.hbs:60]: hmm... clicking the x on the modal does not work for me

Comment (cameronperera) [reply ui/desktop/app/templates/scopes.hbs:60]: Yes that is expected. We decided to keep the modal from being dismissed. The only way to "close" it is to shut down the entire desktop client or navigate to the `/cluster-url` route by clicking the new button we added.

**Review by lisbet-alvarez — APPROVED** [review]
🙌

**Pull request merged**